src/stories/containers/global-activity/global-activity.tsx

```diff
@@ -20,8 +20,8 @@ interface Props {
 }
 
 export default ({ coreUnits }: Props) => {
-  const isLight = useThemeContext().themeMode === 'light';
-
+  const { themeMode } = useThemeContext();
+  const isLight = themeMode === 'light';
```

Comment:
get this from the theme context hook

Reply:
You are right , right now , we need to doing step by step, because is used this way in all the app